awx/api/views.py

```diff
@@ -53,18 +53,21 @@
 # Python Social Auth
 from social_core.backends.utils import load_backends
 
+# Django OAuth Toolkit
+from oauth2_provider.models import get_access_token_model
+
 import pytz
 from wsgiref.util import FileWrapper
 
 # AWX
 from awx.main.tasks import send_notifications, handle_ha_toplogy_changes
 from awx.main.access import get_user_queryset
 from awx.main.ha import is_ha_environment
-from awx.api.authentication import TokenGetAuthentication
+# from awx.api.authentication import TokenGetAuthentication
```

Comment:
If there's code that's just no longer in use like this, let's remove it instead of commenting it out - version control can give us history if we need it to.